test/common/odp_cunit_common.c

```diff
@@ -187,6 +187,12 @@ int odp_cunit_thread_create(int num, int func_ptr(void *), void *const arg[], in
 	odph_thread_common_param_t thr_common;
 	odph_thread_param_t thr_param[num];
 
+	if (num > ODP_THREAD_COUNT_MAX) {
+		fprintf(stderr, "error: %s: too many threads: max=%d\n", __func__,
```

Comment:
The attempted amount could be printed as well.

Reply:
Fixed in V2.